vsphere/README.md

```diff
@@ -0,0 +1,108 @@
+# vSphere Mixin
+
+The vSphere mixin is a set of configurable Grafana dashboards and alerts.
+
+The vSphere mixin contains the following dashboards:
+
+- vSphere cluster
+- vSphere overview
+- vSphere hosts
+- vSphere virtual machines
+- vSphere logs
+
+and the following alerts:
+
+- vSphereHostInfoCPUUtilization
+- vSphereHostWarningMemoryUtilization
+- vSphereDatastoreWarningDiskUtilization
+- vSphereDatastoreCriticalDiskUtilization
+- vSphereHostWarningHighPacketErrors
+
+## vSphere Cluster
+
+The vSphere cluster dashboard provides details on cluster CPU and memory while giving a high level view into hosts, and VM's.
+
+![vSphere Overview Dashboard]()
```

Comment:
```suggestion
![vSphere cluster dashboard]()
```

Reply:
I'm suggesting changing the capitalization to sentence case to match how the dashboard is named in Grafana.